netebpfext/net_ebpf_ext.c

```diff
@@ -566,46 +556,48 @@ net_ebpf_extension_initialize_wfp_components(_Inout_ void* device_object)
 
     status = FwpmSubLayerAdd(_fwp_engine_handle, &ebpf_hook_sub_layer, NULL);
     if (!NT_SUCCESS(status)) {
-        KdPrintEx(
-            (DPFLTR_IHVDRIVER_ID, DPFLTR_INFO_LEVEL, "NetEbpfExt: FwpmSubLayerAdd failed with error %.2X\n", status));
+        NET_EBPF_EXT_LOG_NTSTATUS_API_FAILURE(NET_EBPF_EXT_TRACELOG_KEYWORD_ERROR, "FwpmSubLayerAdd", status);
         goto Exit;
     }
 
     for (index = 0; index < EBPF_COUNT_OF(_net_ebpf_ext_wfp_callout_state); index++) {
         status = _net_ebpf_ext_register_wfp_callout(&_net_ebpf_ext_wfp_callout_state[index], device_object);
         if (!NT_SUCCESS(status)) {
-            KdPrintEx(
-                (DPFLTR_IHVDRIVER_ID,
-                 DPFLTR_INFO_LEVEL,
-                 "NetEbpfExt: _net_ebpf_ext_register_wfp_callout failed for %S with "
-                 "error %.2X\n",
-                 _net_ebpf_ext_wfp_callout_state[index].name,
-                 status));
+            char buf[80];
```

Comment:
TraceLogging APIs permit formatting when the event is consumed. Please extend this macro to accept the additional string.

Reply:
fixed by making this (and another similar) macro variadic.